contracts/drafts/SignatureBouncer.sol

```diff
@@ -75,36 +75,36 @@ contract SignatureBouncer is SignerRole {
      * @dev is the signature of `this + sender` from a signer?
      * @return bool
      */
-    function _isValidSignature(address account, bytes memory signature) internal view returns (bool) {
-        return _isValidDataHash(keccak256(abi.encodePacked(address(this), account)), signature);
+    function _isValidSignature(address sender, bytes memory signature) internal view returns (bool) {
```

Comment:
I'd go the other way around with this one: the signer isn't necessarily the sender of the message, so I think the more generic `account` fits better. What we should do is update the comments so that they no longer read 'sender'.

Reply:
No problem - I'm really excited to start contributing to the ecosystem! Thanks for being so welcoming 😀 

With respect to the `SignatureBouncer`, if I've understood the contract correctly, the signer isn't necessarily the sender but the `account` variable is (at least in the base contract). The `onlyValidSignature[And..]` modifiers are ensuring that the current message sender has been authorised to call the function by some unspecified signer. 

On the other hand, I could imagine derived contracts that use the `_isValidSignature[And..]` set of functions to ensure that a signer has authorised a third party to perform an action on the contract, so your instinct to avoid tightly coupling the function to its current use makes a lot of sense. Happy to follow your lead.

I've updated the PR accordingly  😄

Reply:
Yes, that is precisely the reason why - those functions could even be abstracted away to some sort of signature utilities contract that `SignatureBouncer` uses.